File: demo/src/main/java/com/example/demo/dto/DramaResponse.java
package com.example.demo.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class DramaResponse {
    private String title;
    private Long userId;
    private String userName;
    private String actors;
    private String tag;
    private boolean shown;
    private Float grade;
    private String viewPoint;
    private String link1;
    private String link2;
    private String link3;
    private String posterPath;
    private String category;
    private LocalDateTime updatedAt;
}

File: app/app/src/main/java/front/app/model/DramaResponse.kt
package front.app.model

data class DramaResponse(
    val title: String,
    val userId: Long,
    val userName: String,
    val actors: String? = null,
    val tag: String? = null,
    val shown: Boolean = true,
    val grade: Float? = null,
    val viewPoint: String? = null,
    val link1: String? = null,
    val link2: String? = null,
    val link3: String? = null,
    val updatedAt: String? = null
)
